app/containers/MassEnergizeSuperAdmin/Actions/AllActions.js

```diff
@@ -65,162 +65,168 @@ class AllActions extends React.Component {
     const newData = allActions.filter(
       (a) => (a.community && a.community.id === id) || a.is_global
     );
-    await this.setStateAsync({ data: this.fashionData(newData) });
+    await this.setStateAsync({ data: fashionData(newData) });
   };
 
+  getColumns = () => {
+    const { classes } = this.props;
+    return [
+      {
+        name: "Image",
+        key: "image",
+        options: {
+          filter: false,
+          download: false,
+          customBodyRender: (d) => (
+            <div>
+              {d.image && (
+                <Avatar
+                  alt={d.initials}
+                  src={d.image.url}
+                  style={{ margin: 10 }}
+                />
+              )}
+              {!d.image && <Avatar style={{ margin: 10 }}>{d.initials}</Avatar>}
+            </div>
+          ),
+        },
+      },
+      {
+        name: "Title",
+        key: "title",
+        options: {
+          filter: false,
+          filterType: "textField",
+        },
+      },
+      {
+        name: "Rank",
+        key: "rank",
+        options: {
+          filter: false,
+          customBodyRender: (d) => {
+            return (
+              d && (
+                <TextField
+                  style={{ width: 50, textAlign: "center" }}
+                  key={d.id}
+                  required
+                  name="rank"
+                  variant="outlined"
+                  onChange={async (event) => {
+                    const { target } = event;
+                    if (!target) return;
+                    const { name, value } = target;
+                    if (isNotEmpty(value) && value !== String(d.rank)) {
+                      await apiCall("/actions.rank", {
+                        action_id: d && d.id,
+                        [name]: value,
+                      });
+                    }
+                  }}
+                  label="Rank"
+                  InputLabelProps={{
+                    shrink: true,
+                    maxwidth: "10px",
+                  }}
+                  defaultValue={d.rank}
+                />
+              )
+            );
+          },
+        },
+      },
+      {
+        name: "Tags",
+        key: "tags",
+        options: {
+          filter: true,
+          filterType: "textField",
+        },
+      },
+      {
+        name: "Community",
+        key: "community",
+        options: {
+          filter: true,
+          filterType: "multiselect",
+        },
+      },
+      {
+        name: "Live?",
+        key: "is_live",
+        options: {
+          filter: true,
+          customBodyRender: (d) => {
+            return (
+              <Chip
```

Comment:
I like the <Chip graphic.  There was a request for being able to Live/Unlive from the AllActions page (and other pages).   Can we make this chip respond to a click and quick dialog (Make this action go live?" Yes/cancel?

Reply:
Noted